.github/workflows/code-review.yml

```
@@ -0,0 +1,38 @@
+name: Gemini AI Code Review
+
+on:
+  pull_request_target:
+    types: [labeled]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  gemini-code-review:
+    runs-on: ubuntu-latest
+    if: github.event.label.name == 'gemini-review'
```

Maybe the below to keep our effort low/the-same?

```suggestion
    if: github.event.label.name == 'approved' || github.event.label.name == 'ok-to-test'
```